docs-src/go/how-to-schedule-a-workflow-in-go.md

```diff
@@ -0,0 +1,233 @@
+
+```go
+// Delete the schedule once the sample is done
+defer func() {
+	log.Println("Deleting schedule", "ScheduleID", scheduleHandle.GetID())
+	err = scheduleHandle.Delete(ctx)
+	if err != nil {
+		log.Fatalln("Unable to delete schedule", err)
+	}
+}()
+```
+
+<!--SNIPEND-->
+
+## Describe
+
+You can retrieve information about the current Schedule configuration by using the `describe` action.
+This is helpful when you want to get a detailed view of the Schedule and its associated [Workflow Runs](/concepts/what-is-a-run-id).
+
+To describe a Schedule in Go, use `Describe()` on `scheduleHandle`.
+
+<!--SNIPSTART samples-go-schedule-unpause-describe {"selectedLines": ["115-118"]}-->
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+description, err := scheduleHandle.Describe(ctx)
+if err != nil {
+	log.Fatalln("Unable to describe schedule", err)
+}
+```
+
+<!--SNIPEND-->
+
+## List
+
+When used, `list` returns all available Schedules and their Schedule Ids.
+
+To retrieve a list of Schedules, use `ScheduleClient().List()`.
+
+## Pause/unpause
+
+The `pause` and `unpause` actions enable starting or stopping all future Workflow Runs on a given Schedule.
+Pausing a Schedule halts all future Workflow Runs; unpausing a Schedule allows the Workflow to execute as planned.
+
+Pausing can be enabled when you create a Schedule by setting `State.Paused` to `true`.
+
+<!--SNIPSTART samples-go-schedule-update {"selectedLines": ["90-91"]}-->
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+// Start the schedule paused to demonstrate how to unpause a schedule
+schedule.Description.Schedule.State.Paused = true
+```
+
+<!--SNIPEND-->
+
+Pausing can also be done by using `Pause()` on `scheduleHandle`.
+
+```go
+scheduleHandle.Pause()
+```
+
+To unpause a Schedule, use `Unpause()` on `scheduleHandle`.
+
+<!--SNIPSTART samples-go-schedule-unpause-describe {"selectedLines": ["107-110"]}-->
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+// Unpause schedule
+log.Println("Unpausing schedule", "ScheduleID", scheduleHandle.GetID())
+err = scheduleHandle.Unpause(ctx, client.ScheduleUnpauseOptions{})
+if err != nil {
+	log.Fatalln("Unable to unpause schedule", err)
+}
+```
+
+<!--SNIPEND-->
+
+## Trigger
+
+Triggering a Schedule immediately executes an Action defined in the Schedule.
+By default, `trigger` is subject to the Overlap Policy.
+
+To trigger a Scheduled Workflow Execution in Go, use `Trigger()` on `scheduleHandle`.
+Set desired changes in `ScheduleTriggerOptions`.
+
+<!--SNIPSTART samples-go-schedule-trigger {"selectedLines": ["55-57"]}-->
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+	// Manually trigger the schedule once
```

Comment:
Inconsistent indentation on snippets (some of these are left aligned, some indented)